src/components/Book.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+const Book = ({ book }) => (
+  <tr>
+    <td>{ book.id }</td>
+    <td>{ book.title }</td>
+    <td>{ book.category }</td>
+  </tr>
+);
+
+Book.propTypes = {
+  book: propTypes.objectOf(propTypes.book).isRequired,
```

Comment:
- [ ] This gives an error in the console.
![Screenshot from 2020-10-09 18-18-26](https://user-images.githubusercontent.com/4246637/95607698-686a5480-0a5c-11eb-801d-65ba13961f5c.png)

To validate an object use `shape`. Check [this](https://reactjs.org/docs/typechecking-with-proptypes.html) and [this](https://stackoverflow.com/questions/45764746/react-proptypes-objectof-vs-shape)